cardano-cli/src/Cardano/CLI/Compatible/Helper.hs

```diff
@@ -0,0 +1,131 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RankNTypes #-}
+
+module Cardano.CLI.Compatible.Helper
+  ( mkTxOut
+  , toTxOutInAnyEra
+  )
+where
+
+import Cardano.Api
+import Cardano.Api.Shelley
+
+import Cardano.CLI.Compatible.Exception
+import Cardano.CLI.EraBased.Script.Read.Common
+import Cardano.CLI.Orphan ()
+import Cardano.CLI.Read
+import Cardano.CLI.Type.Common
+
+import Data.Text (Text)
+
+toTxOutInAnyEra
```

Comment:
Regarding the name, isn't that the other way around? If https://github.com/input-output-hk/cardano-node-wiki/pull/74 gets accepted, this should be called: `txOutInAnyEraToTxOut` or `fromTxOutInAnyEra`.

Reply:
It's poorly named but I want to wait till after 74 gets adopted before making the change.